tests/ClangSharp.PInvokeGenerator.UnitTests/FunctionDeclarationDllImportTest.cs

```diff
@@ -69,6 +69,32 @@ public static partial class Methods
             await ValidateGeneratedBindingsAsync(inputContents, expectedOutputContents);
         }
 
+        [Theory]
+        [InlineData("MyStruct<int>", @"MyStruct<int>", "")]
+        [InlineData("MyStruct<bool>", @"[NativeTypeName(""MyStruct<bool>"")] MyStruct<byte>", "")]
```

Comment:
What if the type appears in a non P/Invoke signature?

Reply:
Added a test for this.